src/shared/components/ShareWithColleagues/Modals/DeleteShareUserModal.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import useTranslation from '../../../hooks/useTranslation';
+import ConfirmModal from '../../ConfirmModal/ConfirmModal';
+import { DeleteShareUserModalContent, ShareUserInfo } from '../ShareWithColleagues.types';
+
+type DeleteShareUserModalProps = {
+	isOpen: boolean;
+	onConfirm: () => void;
+	onClose: () => void;
+	user: ShareUserInfo;
+	currentUser: ShareUserInfo;
+};
+
+const DeleteShareUserModal: FC<DeleteShareUserModalProps> = ({
+	isOpen,
+	onConfirm,
+	onClose,
+	user,
+	currentUser,
+}) => {
```

Comment:
i would use the confirmModal directly on the shareWithCollegues component
This wrapper is quite specific and only deals with some translations:

current code:
```jsx
<DeleteShareUserModal
	user={toDeleteShareUser as ShareUserInfo}
	currentUser={currentUser}
	isOpen={isDeleteUserModalOpen}
	onConfirm={handleConfirmDeleteUser}
	onClose={() => setIsDeleteUserModalOpen(false)}
/>
```

using the confirmModal directly:
```jsx
<ConfirmModal
	title={tText('shared/components/share-with-colleagues/modals/delete-share-user-modal___opdracht-verlaten')}
	body={tText('shared/components/share-with-colleagues/modals/delete-share-user-modal___ben-je-zeker-dat-deze-opdracht-wilt-verlaten-als-kijker-deze-actie-kan-niet-ongedaan-gemaakt-worden')}
	confirmLabel={tText('shared/components/share-with-colleagues/modals/delete-share-user-modal___verlaat-opdracht'	)}
	isOpen={isDeleteUserModalOpen}
	confirmCallback={handleConfirmDeleteUser}
	onClose={() => setIsDeleteUserModalOpen(false)}
/>
```

It's so little extra code that it doesn't make sense to make an extra component for this.

Reply:
I made it a separate component because of the translations. The content of the popup depends on who the user is deleting, himself or in case if he is an owner someone else.